assets/js/extensions/shipping-methods/pickup-location/general-settings.tsx

```diff
@@ -78,10 +78,19 @@ const GeneralSettings = () => {
 						'Enable local pickup',
 						'woo-gutenberg-products-block'
 					) }
-					help={ __(
-						'When enabled, local pickup will appear as an option on the block based checkout.',
-						'woo-gutenberg-products-block'
-					) }
+					help={
+						<div>
+							{ __(
+								'When enabled, local pickup will appear as an option on the block based checkout.',
+								'woo-gutenberg-products-block'
+							) }
+							<br />
+							{ __(
```

Comment:
I don't think we need to render this unless they actually use the option. Its low usage, and this is just confusing if you're not familiar with wc settings.

Reply:
Done